File: code-recipes/php/test/b2bdeliveryexperience/BusinessDeliveryExperienceRecipeTest.php
<?php
namespace Test\b2bdeliveryexperience;

use Src\b2bdeliveryexperience\BusinessDeliveryExperienceRecipe;
use Test\util\RecipeTestCase;

class BusinessDeliveryExperienceRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new BusinessDeliveryExperienceRecipe();
        $this->responses = [
            "orders-getOrder",
            "orders-getOrderBuyerInfo",
            "orders-getOrderItems",
            "orders-getOrderAddress",
            "orders-confirmShipment",
        ];
    }
}

File: code-recipes/java/src/test/java/b2bdeliveryexperience/BusinessDeliveryExperienceRecipeTest.java
package b2bdeliveryexperience;

import util.RecipeTest;

import java.util.List;

public class BusinessDeliveryExperienceRecipeTest extends RecipeTest {

    protected BusinessDeliveryExperienceRecipeTest() {
        super(
                new BusinessDeliveryExperienceRecipe(),
                List.of(
                        "orders-getOrder",
                        "orders-getOrderBuyerInfo",
                        "orders-getOrderItems", 
                        "orders-getOrderAddress",
                        "orders-confirmShipment",
                        ""
                )
        );
    }
}